### PR TITLE
Add ComicEarthstar

### PR DESCRIPTION
**Pull request by MikeZeDev**
* CoreView + GraphQL

### CODE REVIEW

File: web/src/engine/FetchProviderNodeWebkit.ts [hunk 1]
```diff
@@ -158,10 +158,10 @@ export async function FetchCSS<T extends HTMLElement>(request: FetchRequest, que
     return [...dom.querySelectorAll(query)] as T[];
 }
 
-export async function FetchGraphQL<TResult>(request: FetchRequest, operationName: string, query: string, variables: string): Promise<TResult> {
+export async function FetchGraphQL<TResult>(request: FetchRequest, operationName: string, query: string, requestVariables: string = undefined): Promise<TResult> {
 
     const graphQLRequest = new FetchRequest(request.url, {
-        body: JSON.stringify({ operationName, query, variables }),
+        body: JSON.stringify({ operationName, query, variables: requestVariables ? requestVariables: {} }),
```
Comment (ronny1982) [inline web/src/engine/FetchProviderNodeWebkit.ts:164]: Type mismatch, `variables` is defined as string, but an empty object `{}` is used as default.

File: web/src/engine/websites/ComicEarthStar.ts [hunk 2]
```diff
@@ -0,0 +1,121 @@
+import { Tags } from '../Tags';
+import icon from './ComicEarthStar.webp';
+import { DecoratableMangaScraper, Manga, type MangaPlugin } from '../providers/MangaPlugin';
+import * as CoreView from './decorators/CoreView';
+import * as Common from './decorators/Common';
+import { FetchGraphQL, FetchRequest } from '../FetchProvider';
+
+const queryOneshots = `
+            query Earthstar_Oneshot {
```
Comment (ronny1982) [inline web/src/engine/websites/ComicEarthStar.ts:9]: Indent should be 4 spaces, but is 8 spaces

File: web/src/engine/websites/ComicEarthStar.ts [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+                seriesOneshot: serialGroup(groupName: "連載・読切：読切作品") {
+                    seriesSlice {
+                        seriesList {
+                            ...Earthstar_SeriesListItem_Series
+                        }
+                    }
+                }
+            }
+            fragment Earthstar_SeriesListItem_Series on Series {
```
Comment (ronny1982) [inline web/src/engine/websites/ComicEarthStar.ts:18]: Fragment is only used once, consider to embed the content directly and drop the fragment

File: web/src/engine/FetchProviderNodeWebkit.ts (hunk 1 above)
Comment (MikeZeDev) [reply web/src/engine/FetchProviderNodeWebkit.ts:164]: IKR .If i use a string the request doesnt work. Teh request
Do you suggest i modify FetchGraphQL argument to pass an objet of type "unknown" so its settled ?

Comment (MikeZeDev) [reply web/src/engine/FetchProviderNodeWebkit.ts]: Changes reverted, and so its gonna need #406 merged to work :)

**Review by ronny1982 — APPROVED** [review]

**Pull request merged**